src/pages/Profile/index.jsx

```diff
@@ -68,27 +65,27 @@ export default function Profile() {
     event.preventDefault();
     switch (status) {
       case "wishlist": {
+        if (dropStatus === "wishlist") break;
         setWishlist(wishlist.filter((item) => item.id !== data.id));
-        transferGame(status, dropStatus);
+        transferGame(dropStatus);
         break;
       }
       case "playing": {
+        if (dropStatus === "playing") break;
```

Comment:
why case and if together?

Reply:
there are two conditions status and dropStatus. I am checking if the status and drop status is same then skip it.

Reply:
fixed it